### PR TITLE
Rerunning msbroker should not fail

### PR DESCRIPTION
**Pull request by sedroche**
Re running the msbroker playbook was failing. Added some fixes.

### CODE REVIEW

File: evals/roles/msbroker/tasks/main.yml [hunk 1]
```diff
@@ -27,17 +27,34 @@
   shell: oc new-project {{ msbroker_namespace }}
   when: msbroker_check_namespace_exists.rc != 0
 
-- name: Check namespace for existing resources
-  shell: oc get all -n {{ msbroker_namespace }}
-  register: msbroker_resources_exist
+- name: Ensure CRDs do not exist
+  shell: "oc delete -f {{ item }}"
```
Comment (pmccarthy) [inline evals/roles/msbroker/tasks/main.yml]: Do we always want to remove these? In its current state every re-run of the installer will remove and re-create the CRDs

Comment (pmccarthy) [reply evals/roles/msbroker/tasks/main.yml]: Same applies to the clusterrole, clusterrolebinding and the clusterservicebroker

Comment (maleck13) [reply evals/roles/msbroker/tasks/main.yml]: I think these are ok to remove and update each time as they would be removed pretty much straight away (IE there is  not running pods etc to wait on)

Comment (sedroche) [reply evals/roles/msbroker/tasks/main.yml]: Yep, I only delete/create on resources that are quickly deleted/created. The deployment is still checked to see if it exists.

**Pull request merged**